README.md

```diff
@@ -26,6 +26,13 @@ You can learn more by checking out the [docs](docs/README.md)
 
 [Install](./docs/install.md) Triggers, check out the [installation guide](./docs/install.md),  [examples](./examples/README.md) or follow the [getting started guide](./docs/getting-started/README.md) to become familiar with the project. The getting started guide walks through setting up an end-to-end image building solution, which will be triggered from GitHub `push` events.
 
+### Read the docs
+
+| Version | Docs | Examples | Getting Started |
+| ------- | ---- | -------- | --------------- |
+| [HEAD](DEVELOPMENT.md#install-pipeline) | [Docs @ HEAD](/docs/README.md) | [Examples @ HEAD](/examples) | [Getting Started @ HEAD](/docs/getting-started#getting-started-with-triggers) |
```

Comment:
I think that this should be absolute links (`https://github.com/tektoncd/triggers/blob/master/DEVELOPMENT.md#install-pipeline`) instead of relative links (`DEVELOPMENT.md#install-pipeline`), because after our next release, when someone looks back at the `v0.2.0` docs, if they want to navigate back to `HEAD`, then this link would not work properly. It would keep them on the `v0.2.0` branch docs.

Reply:
Ahh! Nice catch!